src/main/java/com/nylas/EventConferencing.java

```diff
@@ -0,0 +1,86 @@
+package com.nylas;
+
+import java.util.List;
+import java.util.Map;
+
+public class EventConferencing {
```

Comment:
I suggest moving this to a nested class in Events called Conferencing similar to other Event subobjects